### ISSUE TITLE
CHANGELOG.md

### ISSUE THREAD

**Issue by armab** [bitovi/bitops#222]
A bit relevant to https://github.com/bitovi/bitops/issues/189 and https://github.com/bitovi/bitops/discussions/215.

https://github.com/bitovi/bitops/releases is really nicely composed and simple 👍 

It might be helpful when the external community would start contributing more to do the same with https://keepachangelog.com/, so every new PR adds a record in `CHANGELOG.md` as we go.

@bitops/maintainers `v2` or any other preference?

**Comment by mickmcgrath13** [comment]
v2 seems like a great time to start.  @armab what's the LOE look like to set this up?

**Comment by armab** [comment]
Really easy one, low priority, more of the consensus question.
Just unbufferring random ideas really quick instead of jumping straight into it.
I'd also add some GH Actions integration to make sure the CHANGELOG record is enforced as part of the CI (including the forked builds).

**Comment by PhillypHenning** [comment]
Should I update the name of this [page](https://github.com/bitovi/bitops/blob/plugins/docs/migration/2.0.0-migration.md) to `2.0.0-CHANGELOG.md` ??

**Comment by armab** [comment]
No, it's good. The migration page looks absolutely right for specific steps someone needs to take version-to-version, not to confuse with the Changelog.
